rpm/qtlocation.spec

```diff
@@ -147,6 +159,14 @@ find %{buildroot}%{_libdir}/pkgconfig -type f -name '*.pc' \
 find %{buildroot}%{_libdir} -type f -name '*.prl' \
 -exec sed -i -e "/^QMAKE_PRL_BUILD_DIR/d;s/\(QMAKE_PRL_LIBS =\).*/\1/" {} \;
 
+#install docs
+mkdir -p %{buildroot}/%{_docdir}/qtlocation/
+mkdir -p %{buildroot}/%{_docdir}/qtpositioning/
+cp -R doc/qtlocation/* %{buildroot}/%{_docdir}/qtlocation/
+cp doc/qtlocation.qch %{buildroot}/%{_docdir}/qtlocation/
+cp -R doc/qtpositioning/* %{buildroot}/%{_docdir}/qtpositioning/
+cp doc/qtpositioning.qch %{buildroot}/%{_docdir}/qtpositioning/
+
```

Comment:
Please use install here or change the permissions later to make sure the permissions are good,

Reply:
The permissions are fine